operations/system_management_health/Troubleshoot_Grafana_Dashboard.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshoot Grafana Dashboard
```

Comment:
A bullet point for this new file should be added to the `operations/index.md#system-management-health` file